engines/rsdkv3/build.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+sudo apt-get install build-essential git libsdl2-dev libvorbis-dev libogg-dev libtheora-dev libglew-dev libgbm-dev libdrm-dev
```

Comment:
These shouldn't be needed with the steam runtime docker image. If you need to test again, feel free to use the docker image called ```registry.gitlab.steamos.cloud/steamrt/soldier/sdk:0.20220824.0```

Preferably, if there are any packages that are not provided by the steam runtime (I believe most if not all of those should be supported), then they should be built since there isn't a guarantee of a package being part of the system.